cms/djangoapps/contentstore/rest_api/v1/serializers/common.py

```diff
@@ -17,3 +17,59 @@ class CourseCommonSerializer(serializers.Serializer):
     rerun_link = serializers.CharField()
     run = serializers.CharField()
     url = serializers.CharField()
+
+
+# class StrictSerializer(serializers.Serializer):
```

Comment:
This duplicate (commented) class definition should be removed

Reply:
done